cluster/gce/util.sh

```diff
@@ -156,8 +156,40 @@ function copy-if-not-staged() {
   fi
 }
 
+# Prepare a tarball of kube-system manifests for trusty based cluster.
+#
+# Vars set:
+#   KUBE_MANIFESTS_TAR_URL
+#   KUBE_MANIFESTS_TAR_HASH
+function prepare-manifests-tar() {
```

Comment:
This seems reasonable for the developer workflow, but for releases we will want to do something similar with the build scripts so that official releases have the manifest files available too.